xml/release-notes.xml

```diff
@@ -513,31 +513,23 @@
 
  </sect1>
 
- <!-- <sect1 xml:id="sec.driver">
+ <sect1 xml:id="sec.driver">
   <title>Drivers and Hardware</title>
   <para/> -->
 
-  <!-- <sect2 xml:id="sec.driver.nvidia-hybrid">
-   <!-/- boo#1094834 -/->
-   <title>Hang on Machines with Nvidia GPUs and Hybrid Graphics</title>
+  <sect2 xml:id="sec.driver.sig">
+   <!-/- bsc#173158 -/->
+   <title>CONFIG_MODULE_SIG=y</title>
```

Comment:
This heading does not say anything useful for people not being familiar with the technical details: I suggest something more speaking like "Secure Boot: 3rd Party drivers need to be properly signed".
The CONFIG_MODULE_SIG=y setting then should be mentioned  below (maybe together with an explanation on how to turn it off (and what that would imply).